src/lib/Session.svelte

```diff
@@ -134,6 +134,11 @@
     encrypt = await Encrypt.new(key);
     const encryptedZeros = await encrypt.zeros();
 
+    const searchParams = new URLSearchParams(window.location.search);
+    if (searchParams.get("hide_key") === "true") {
+      window.location.hash = "";
```

Comment:
note: this will break if `onMount` runs more than once here

i'm operating under the assumption that it only happens once; would appreciate insight